core/src/components/textarea/test/size/textarea.e2e.ts

```diff
@@ -6,6 +6,75 @@ import { configs, test } from '@utils/test/playwright';
  */
 configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
   test.describe(title('textarea: size'), () => {
+    test.describe('textarea: size small', () => {
+      test('should not have visual regressions', async ({ page }) => {
+        await page.setContent(
+          `
+          <ion-textarea
+            size="small"
+            label="Email"
+            value="hi@ionic.io"
+          ></ion-textarea>
+        `,
+          config
+        );
+
+        const textarea = page.locator('ion-textarea');
+        await expect(textarea).toHaveScreenshot(screenshot(`textarea-size-small`));
+      });
+      test('should render correctly with stacked label', async ({ page }) => {
+        await page.setContent(
+          `
+          <ion-textarea
+            size="small"
+            label="Email"
+            label-placement="stacked"
+            value="hi@ionic.io"
+          ></ion-textarea>
+        `,
+          config
+        );
+
+        const textarea = page.locator('ion-textarea');
+        await expect(textarea).toHaveScreenshot(screenshot(`textarea-size-small-label-stacked`));
+      });
+      test('should not have visual regressions with fill outline', async ({ page }) => {
+        await page.setContent(
+          `
+          <ion-textarea
+            size="small"
+            fill="outline"
+            label="Email"
+            label-placement="stacked"
+            value="hi@ionic.io"
+          ></ion-textarea>
+        `,
+          config
+        );
+
+        const textarea = page.locator('ion-textarea');
+        await expect(textarea).toHaveScreenshot(screenshot(`textarea-size-small-outline`));
+      });
+      test('should not have visual regressions with fill outline and round shape', async ({ page }) => {
```

Comment:
This test seems redundant with `should not have visual regressions with fill outline` since the default shape is round. I would vote to remove this one.

Reply:
Removed https://github.com/ionic-team/ionic-framework/pull/29781/commits/06abd9481ffd6a7ae50a6fa9083dad11b46adaa0